Monitoring/zabbix/userparameter_murmur.conf

```diff
@@ -0,0 +1,3 @@
+# Zabbix agent custom keys
+# SPDX-License-Identifier: AGPL-3.0-or-later
```

Comment:
I'm a bit confused what is being described as AGPL here in metadata and in the readme. The readme includes instructions to install munin-murmur.py, which is a different license.

The yaml configuration you are contributing is licensed under AGPL?

Is declaring that in the SPDX-License-Identifier sufficient? What is it addressing/describing?